modules/tinymce/src/core/demo/html/full_demo.html

```diff
@@ -11,32 +11,55 @@
+          </tr>
+          <tr style="height: 36.1953px;">
+          <td>1</td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></span></td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;">red and blue text</span></span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;"><span style="color: #b96ad9;"><span style="color: #2dc26b;">red, blue, purple, green text</span></span></span></span></td>
+          <td><span style="opacity: 0.9;"><span style="border: 1px solid black;"><span style="opacity: 0.8;"><span style="border: 1px solid green;">nested wrapped</span></span></span></span></td>
+          </tr>
+          <tr style="height: 36.1953px;">
+          <td>2</td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></span></span></td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;">red and blue text</span></span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;"><span style="color: #b96ad9;"><span style="color: #2dc26b;">red, blue, purple, green text</span></span></span></span></td>
+          <td><span style="opacity: 0.9;"><span style="border: 1px solid black;"><span style="opacity: 0.8;"><span style="border: 1px solid green;">nested wrapped</span></span></span></span></td>
+          </tr>
+          <tr style="height: 36.1953px;">
+          <td>3</td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;"><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></span></span></span></td>
+          <td><span style="color: #843fa1; background-color: #2dc26b;">purple on green text</span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;">red and blue text</span></span></td>
+          <td><span style="color: #e03e2d;"><span style="color: #236fa1;"><span style="color: #b96ad9;"><span style="color: #2dc26b;">red, blue, purple, green text</span></span></span></span></td>
+          <td><span style="opacity: 0.9;"><span style="border: 1px solid black;"><span style="opacity: 0.8;"><span style="border: 1px solid green;">nested wrapped</span></span></span></span></td>
+          </tr>
+          </tbody>
+        </table>
+      </textarea>
```

Comment:
This file should be reverted

Reply:
Moved these changes to a separate demo. Changes in FullDemo are reverted.